chart/values.yaml

```diff
@@ -6,6 +6,22 @@ image_repository: quay.io
 image_name: pet-battle-tournament
 image_namespace: eformat
 image_version: latest
```

Comment:
Need to move away from latest

Reply:
yeah .. cool .. BUT i'm a trunk based development type of guy ;) 🤠🤠🤠

give me latest cause is has all the coolnees and fixes !!

see above .. we can tag it etc in quay.io